integration/form-test.ts

```diff
@@ -576,3 +576,64 @@ test.describe("Forms", () => {
     });
   });
 });
+
+test.describe("Forms in an app with custom routes", () => {
+  let fixture: Fixture;
+  let appFixture: AppFixture;
+
+  test.beforeAll(async () => {
+    fixture = await createFixture({
+      files: {
+        "remix.config.js": `
```

Comment:
Make sure to use our helper function, for better IDE code highlighting

```suggestion
        "remix.config.js": js`
```